Build error after creating a new product following the guide

#### Description of problem:
I set up the build environment and clone this repo. Created a new product following this guide: https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html. But I got an error when trying to build it.
```
FAILED: custom6/ssg_build_compile_all-custom6
cd /root/haoxiang/git_project/content/build/custom6 && /usr/bin/cmake -E make_directory /root/haoxiang/git_project/content/build/custom6/profiles && env PYTHONPATH=/root/haoxiang/git_project/content /usr/bin/python3 /root/haoxiang/git_project/content/build-scripts/compile_all.py --resolved-base /root/haoxiang/git_project/content/build/custom6 --project-root /root/haoxiang/git_project/content --build-config-yaml /root/haoxiang/git_project/content/build/build_config.yml --product-yaml /root/haoxiang/git_project/content/build/custom6/product.yml --sce-metadata /root/haoxiang/git_project/content/build/custom6/checks/sce/metadata.json --rule-id off && /usr/bin/cmake -E touch /root/haoxiang/git_project/content/build/custom6/ssg_build_compile_all-custom6
Traceback (most recent call last):
  File "/root/haoxiang/git_project/content/build-scripts/compile_all.py", line 235, in <module>
    main()
  File "/root/haoxiang/git_project/content/build-scripts/compile_all.py", line 207, in main
    load_benchmark_source_data_from_directory_tree(loader, env_yaml, product_yaml)
  File "/root/haoxiang/git_project/content/build-scripts/compile_all.py", line 88, in load_benchmark_source_data_from_directory_tree
    loader.process_directory_trees(relevant_benchmark_sources)
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 2797, in process_directory_trees
    return self.process_directory_tree(start_dir, extra_group_dirs)
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 2781, in process_directory_tree
    self._load_group_process_and_recurse(start_dir)
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 2750, in _load_group_process_and_recurse
    self.loaded_group = self.load_benchmark_or_group(guide_directory)
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 2727, in load_benchmark_or_group
    group = Benchmark.from_yaml(
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 533, in from_yaml
    benchmark.id_ = env_yaml["benchmark_id"]
KeyError: 'benchmark_id'
ninja: build stopped: subcommand failed.
```

#### SCAP Security Guide Version:
master branch. Commit hash: 0f151a1b78273764df0d0e86a5088d089b386231

#### Operating System Version:
Ubuntu 20.04.3

#### Steps to Reproduce:

1. Clone and create a new product following the guide.
2. `./build_product custom6`

#### Actual Results:
Build failed
```
FAILED: custom6/ssg_build_compile_all-custom6
cd /root/haoxiang/git_project/content/build/custom6 && /usr/bin/cmake -E make_directory /root/haoxiang/git_project/content/build/custom6/profiles && env PYTHONPATH=/root/haoxiang/git_project/content /usr/bin/python3 /root/haoxiang/git_project/content/build-scripts/compile_all.py --resolved-base /root/haoxiang/git_project/content/build/custom6 --project-root /root/haoxiang/git_project/content --build-config-yaml /root/haoxiang/git_project/content/build/build_config.yml --product-yaml /root/haoxiang/git_project/content/build/custom6/product.yml --sce-metadata /root/haoxiang/git_project/content/build/custom6/checks/sce/metadata.json --rule-id off && /usr/bin/cmake -E touch /root/haoxiang/git_project/content/build/custom6/ssg_build_compile_all-custom6
Traceback (most recent call last):
  File "/root/haoxiang/git_project/content/build-scripts/compile_all.py", line 235, in <module>
    main()
  File "/root/haoxiang/git_project/content/build-scripts/compile_all.py", line 207, in main
    load_benchmark_source_data_from_directory_tree(loader, env_yaml, product_yaml)
  File "/root/haoxiang/git_project/content/build-scripts/compile_all.py", line 88, in load_benchmark_source_data_from_directory_tree
    loader.process_directory_trees(relevant_benchmark_sources)
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 2797, in process_directory_trees
    return self.process_directory_tree(start_dir, extra_group_dirs)
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 2781, in process_directory_tree
    self._load_group_process_and_recurse(start_dir)
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 2750, in _load_group_process_and_recurse
    self.loaded_group = self.load_benchmark_or_group(guide_directory)
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 2727, in load_benchmark_or_group
    group = Benchmark.from_yaml(
  File "/root/haoxiang/git_project/content/ssg/build_yaml.py", line 533, in from_yaml
    benchmark.id_ = env_yaml["benchmark_id"]
KeyError: 'benchmark_id'
ninja: build stopped: subcommand failed.
```

#### Expected Results:
Generate the XML files for custom platform.

#### Additional Information/Debugging Steps:
I think the `benchmark_id` is missing in the `product/custom6/product.yml` file.

--- a/docs/manual/developer/03_creating_content.md
+++ b/docs/manual/developer/03_creating_content.md
@@ -172,6 +172,7 @@ their `product.yml` file using `benchmark_root` key. For example,
     full_name: Red Hat Enterprise Linux 9
     type: platform
 
+    benchmark_id: RHEL-9
     benchmark_root: "../linux_os/guide"
 
     .....
@@ -431,6 +432,7 @@ product: $NEW_PRODUCT
 full_name: $FULL_NAME
 type: platform
 
+benchmark_id: $CAPITAL_NAME-$VERSION
 benchmark_root: "../../linux_os/guide"
 
 components_root: "../../components"

